staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -77,7 +77,7 @@ func NewDefaultFieldManager(models openapiproto.Models, objectConverter runtime.
 	if err != nil {
 		return nil, fmt.Errorf("failed to create field manager: %v", err)
 	}
-	return newDefaultFieldManager(f, objectCreater, kind), nil
+	return newDefaultFieldManager(f, models, objectCreater, kind), nil
```

Comment:
This means will have to parse the models twice, it's probably not the end of the world, but I'm curious if we should do it differently.

Reply:
Hm we could pass a type converter to this field manager and the structured field managers to parse the models once

Reply:
Done

Reply:
Hm actually is this a good idea? TypeConverter is in `internal`